### PR TITLE
Add large technology card to react components

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [x] Documentation provided. (Add links if there are any)
- [x] Unit tests provided. (Add links if there are any)
- [x] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by DimalChandrasiri — APPROVED** [review]

File: modules/react-components/src/components/card/technology-card-large.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+     */
+    raised?: boolean;
+    /**
+     * Resolve i18n tag for feature available content
+     */
+    featureAvailable?: string;
+}
+
+/**
+ * Technology card component.
```
Comment (dasuni-30) [inline modules/react-components/src/components/card/technology-card-large.tsx:81]: Can we update the descriptions?

File: modules/react-components/src/components/card/technology-card-large.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        </div>
+
+    );
+};
+
+/**
+ * Default props for the technology card component.
+ */
+LargeTechnologyCard.defaultProps = {
+    "data-componentid": "technology-card"
```
Comment (dasuni-30) [inline modules/react-components/src/components/card/technology-card-large.tsx:210]:
```suggestion
    "data-componentid": "technology-card-large"
```

**Review by dasuni-30 — APPROVED** [review]

File: modules/react-components/src/components/card/technology-card-large.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply modules/react-components/src/components/card/technology-card-large.tsx:81]: Will fix in a follow-up PR

File: modules/react-components/src/components/card/technology-card-large.tsx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+     */
+    featureAvailable?: string;
+}
+
+/**
+ * Technology card component.
+ *
+ * @param props - Props injected to the components.
+ *
+ * @returns the TechnologyCard component
```
[the remark this answers is not in the document]
Comment (DonOmalVindula) [reply modules/react-components/src/components/card/technology-card-large.tsx:85]: Will fix in a follow-up PR

**Pull request merged**